pkg/auth/requests/authenticate.go

```diff
@@ -137,7 +154,14 @@ func (a *tokenAuthenticator) Authenticate(req *http.Request) (bool, string, []st
 		go a.userAuthRefresher.TriggerUserRefresh(token.UserID, false)
 	}
 
-	return true, token.UserID, groups, nil
+	authResp.IsAuthed = true
+	authResp.User = token.UserID
+	authResp.UserPrincipal = token.UserPrincipal.Name
+	authResp.Groups = groups
+	logrus.Debugf("Auth filter calling provider to get extra info %v", token.AuthProvider)
+	authResp.Extras = providers.GetAttributesForAuditLog(token.AuthProvider, token)
```

Comment:
Is there a possibility this ends up being a nil map? If so we should guard against that